csbuild/tools/common/apple_tool_base.py

```diff
@@ -33,8 +33,8 @@
 
 from abc import ABCMeta
 
-from ..._utils import ordered_set
 from ..._utils.decorators import MetaClass
+from ..._utils.ordered_set import OrderedSet
```

Comment:
This technically violates the coding style, but looking through the code, it seems like I've violated it in a lot of places, so this is just a note that this needs to be cleaned up throughout the codebase. I prefer using the whole namespace when possible because it makes it easier for people unfamiliar with the codebase to understand where things are defined when they see them being used - i.e., it makes it obvious what's defined in this file vs another file, and it also makes it obvious what the file structure is to find the thing they're looking for.